### PR TITLE
Tests for obtaining a list of services

### PR DESCRIPTION
**Pull request by ZdanovichOleg**

### CODE REVIEW

File: helpers/serviceHelper.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const chance = require('chance').Chance();
+import request from 'supertest';
+import 'dotenv/config';
+import * as constants from './constants';
+
+export const serviceData = async() => {
+  return {
+    name: chance.sentence({ words: 2 }),
+    vendor: await constants.vendorId(),
+    clientPrice: chance.prime(),
```
Comment (dashakhan) [inline helpers/serviceHelper.js:10]: Similarly to what Raf pointed out with my code, chance.prime() returns a random prime number between 2 and 10000 by default, which will not cover all scenarios

File: helpers/serviceHelper.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  };
+};
+export function createService(serviceData) {
+  return request(process.env.BASE_URL)
+    .post('service')
+    .set('Authorization', process.env.TOKEN)
+    .send(serviceData);
+}
+
+export function getAllServices(token) {
```
Comment (dumpdusty) [inline helpers/serviceHelper.js:22]: same question - what is a reason to parametrize token in this function only?

**Review by dumpdusty — DISMISSED** [review]
please add PR description next time, rest is good

**Review by dashakhan — APPROVED** [review]

**Review by dumpdusty — APPROVED** [review]
LGTM

**Pull request merged**